client/payment-details/payment-method/index.js

```diff
@@ -29,7 +36,10 @@ const PaymentDetailsPaymentMethod = ( { charge = {}, isLoading } ) => {
 				PaymentMethodDetails = SofortDetails;
 				break;
 		}
+	} else {
+		return <></>;
```

Comment:
Based on my experience usually `return null;` is used to prevent component rendering. Is there a benefit from rendering fragment (`<></>`) instead?

Reply:
Yeah, I swore `false` used to work too, but I got an error with it. I'll try again before introducing this new concept

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/1840/commits/c9b58c63b20bfbb86a8f2888960c2dd21e399a46